frontend/shared/src/components/header/Header.tsx

```diff
@@ -58,14 +55,21 @@ const Header: React.FC<HeaderProps> = ({
     menuOpen,
     toggleMenu,
     closeMenu,
-    handleNavigationItemClick,
     handleLogin,
     handleLogout,
-  } = useHeader(locale, onNavigationItemClick, login);
+  } = useHeader(locale, login);
 
-  const handleTitleClick = onTitleClick
-    ? () => onTitleClick(closeMenu)
-    : undefined;
+  const goToPage = useGoToPage();
+
+  const handleClickLink = React.useCallback(
```

Comment:
In Helsinki Benefit, the title was not clickable (which might change in the future) but with this change, it looks like this is no longer the case, right?

Reply:
ah, ok, I thought it would be the same in every service that it would link to the `/` homepage.

Reply:
I simply remove the the link from the title beause out services neither need it. Let's change it in the future if there is some need for the change